effective date and implementation (#673)
usnistgov/FIPS201: _Frontmatter/announcement.md

# 9. Effective Date. {#f-9}

> This Standard will be effective immediately upon final publication of this revision, superseding FIPS 201-2. The Standard includes new and updated features as well as features that are being deprecated or removed as outlined in the [revision history](../_FIPS201/revision.md).  The effective dates of these features depend upon the release of revised NIST Special Publications or the release of new NIST Special Publications which are targeted to be released within 18 month of publication of this Standard. An enumeration of NIST Special Publications associated with this Standard is provided in [Section 1.4](introduction.md#s-1-4). Per [Section 7](../_FIPS201/announcement.md#s-7), the OMB provides implementation oversight for this Standard. NIST's Special Publications may reflect implementation targets as appropriate or be provided separately by OMB.  

Reviewer: ```suggestion
This Standard will be effective immediately upon final publication of this revision, superseding FIPS 201-2. The Standard includes new and updated features as well as features that are being deprecated or removed as outlined in the [revision history](../_FIPS201/revision.md).  The effective dates of these features depend upon the release of revised NIST Special Publications or the release of new NIST Special Publications that are targeted to be released within 18 months of publication of this Standard. An enumeration of NIST Special Publications associated with this Standard is provided in [Section 1.4](introduction.md#s-1-4). Per [Section 7](../_FIPS201/announcement.md#s-7), the OMB provides implementation oversight for this Standard. NIST's Special Publications may reflect implementation targets as appropriate or be provided separately by OMB.  
```

Reviewer 2: ```suggestion
This Standard will be effective immediately upon final publication of this revision, superseding FIPS 201-2. The Standard includes new and updated features as well as features that are being deprecated or removed as outlined in the revision history in [Appendix E](../_Appendix/revisions.md).  The effective dates of these features depend upon the release of revised NIST Special Publications or the release of new NIST Special Publications which are targeted to be released within 18 month of publication of this Standard. An enumeration of NIST Special Publications associated with this Standard is provided in [Section 1.4](introduction.md#s-1-4). Per [item 7](../_FIPS201/announcement.md#f-7), the OMB provides implementation oversight for this Standard. NIST's Special Publications may reflect implementation targets as appropriate or be provided separately by OMB.  
```

> This Standard will be effective immediately upon final publication of this revision, superseding FIPS 201-2. The Standard includes new and updated features as well as features that are being deprecated or removed as outlined in the revision history in [Appendix E](../_Appendix/revisions.md).  The effective dates of these features depend upon the release of revised NIST Special Publications or the release of new NIST Special Publications that are targeted to be released within 18 months of publication of this Standard. An enumeration of NIST Special Publications associated with this Standard is provided in [Section 1.4](introduction.md#s-1-4). Per [item 7](../_FIPS201/announcement.md#f-7) of this preamble, the OMB provides implementation oversight for this Standard. The implementation schedule may be reflected in NIST's Special Publications or may be provided separately by OMB, as appropriate.

Reviewer 2: ```suggestion
This Standard will be effective immediately upon final publication of this revision, superseding FIPS 201-2. The Standard includes new and updated features as well as features that are being deprecated or removed as outlined in the revision history in [Appendix E](../_Appendix/revisions.md).  The effective dates of these features depend upon the release of revised NIST Special Publications or the release of new NIST Special Publications that are targeted to be released within 18 months of publication of this Standard. An enumeration of NIST Special Publications associated with this Standard is provided in [Section 1.4](introduction.md#s-1-4). Per [item 7](../_FIPS201/announcement.md#f-7) of this preamble, OMB provides implementation oversight for this Standard. The implementation schedule may be reflected in NIST's Special Publications or may be provided separately by OMB, as appropriate.
```

# 9. Effective Date. {#f-9}

> This Standard will be effective immediately upon final publication of this revision, superseding FIPS 201-2. The Standard includes new and updated features as well as features that are being deprecated or removed as outlined in the [revision history](../_FIPS201/revision.md). The effective dates of these features depend upon the release of revised NIST Special Publications or the release of new NIST Special Publications [^listOfPubs] and are targeted to be released within 18 month of release of this Standard. Per [Section 7](../_FIPS201/announcement.md#s-7), the OMB provides implementation oversight for this Standard. NIST's Special Publications may reflect implemention targets as approriate or be provided seperately by OMB.  

Author: ```suggestion
This Standard will be effective immediately upon final publication of this revision, superseding FIPS 201-2. The Standard includes new and updated features as well as features that are being deprecated or removed as outlined in the [revision history](../_FIPS201/revision.md).  The effective dates of these features depend upon the release of revised NIST Special Publications or the release of new NIST Special Publications which are targeted to be released within 18 month of publication of this Standard. An enumeration of NIST Special Publications associated with this Standard is provided in [Section 1.4](introduction.md#s-1-4). Per [Section 7](../_FIPS201/announcement.md#s-7), the OMB provides implementation oversight for this Standard. NIST's Special Publications may reflect implementation targets as appropriate or be provided separately by OMB.  
```
```suggestion
This Standard will be effective immediately upon final publication of this revision, superseding FIPS 201-2. The Standard includes new and updated features as well as features that are being deprecated or removed as outlined in the [revision history](../_FIPS201/revision.md). The effective dates of these features depend upon the release of revised NIST Special Publications or the release of new NIST Special Publications and are targeted to be released within 18 month of release of this Standard. Per [Section 7](../_FIPS201/announcement.md#s-7), the OMB provides implementation oversight for this Standard. NIST's Special Publications may reflect implementation targets as appropriate or it may be provided separately by OMB.  
```

> [^listOfPubs]: See [Section 1.4](introduction.md#s-1-4) for an enumeration of NIST Special Publications associated with this Standard.

Author: ```suggestion
```
